graphql.go

```diff
@@ -244,6 +242,19 @@ type graphResponse struct {
 	Errors []graphErr
 }
 
+func (resp graphResponse) bundleErrors() error {
+	var err error
+
+	for _, graphqlError := range resp.Errors {
+		if err != nil {
```

Comment:
`err` was never initialized here so it will always be nil, right?

Reply:
Yep, in that case it'll go to the else block and initialize with the first error in `resp.Errors`